test/minikube/minikube_kaa_additions_test.go

```diff
@@ -92,3 +92,41 @@ func TestKaaLimitedPermissions(t *testing.T) {
 	require.True(t, len(config.Volumes) == 0)
 	require.True(t, len(config.Pods) == 0)
 }
+
+func TestKaaRolebindingDisabled(t *testing.T) {
+	// This test requires NuoDB 4.2+ or 4.1.2+
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &helm.Options{
+		SetValues: map[string]string{
+			"nuodb.addRoleBinding": "false",
+		},
+	}, 1, "")
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+	testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
+		SetValues: map[string]string{
+			"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+		},
+	})
```

Comment:
do you need the database here?

Reply:
Later on, we are checking that there are no K8s resources returned by `nuocmd`. So that we minimize the test time, I will remove the database and skip the deployments check.